packages/nube-devtools/src/devtools/components/tree-view.tsx

```
@@ -0,0 +1,246 @@
+  isExpanded?: boolean
+}) => {
+  return (
+    <div className="flex items-center">
+      <div
+        onClick={onClick}
+        onKeyDown={(e) => {
+          if (e.key === 'Enter' || e.key === ' ') {
+            onClick()
+          }
+        }}
+        role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)